src/xrCore/xr_shortcut.h

```diff
@@ -17,21 +17,22 @@ struct XRCORE_API xr_shortcut
         {
             u8 key;
             Flags8 ext;
-        };
+        } ext_key;
         u16 hotkey;
     };
 
-    xr_shortcut(u8 k, bool a, bool c, bool s) : key(k)
```

Comment:
Why have you removed this?
```cpp
 : key(k)
```
It's equal to
```cpp
ext_key.key = k;
```

Reply:
This line provides error to.